docs/transactions.md

```diff
@@ -1607,19 +1532,123 @@ Gets all revoked NOC root certificates.
 - REST API:
   - GET `/dcl/pki/revoked-noc-root-certificates`
 
-### GET_REVOKED_NOC_X509_ROOT_CERT
+### Common
+
+#### GET_CERT
 
 **Status: Implemented**
 
-Gets a revoked NOC root certificate by the given subject and subject key ID attributes.
+Gets a certificate (either root or non-root) by the given subject and subject key ID attributes.
+Revoked certificates are not returned.
+Use [GET_ALL_REVOKED_CERTS](#get_all_revoked_certs) to get a list of all revoked certificates.
 
 - Parameters:
-  - subject: `string` - Base64 encoded subject DER sequence bytes of the certificate.
-  - subject_key_id: `string` - Certificate's `Subject Key Id` in hex string format, e.g., `5A:88:0E:6C:36:53:D0:7F:B0:89:71:A3:F4:73:79:09:30:E6:2B:DB`.
+  - subject: `string`  - certificates's `Subject` is base64 encoded subject DER sequence bytes
+  - subject_key_id: `string`  - certificates's `Subject Key Id` in hex string format, e.g: `5A:88:0E:6C:36:53:D0:7F:B0:89:71:A3:F4:73:79:09:30:E6:2B:DB`
 - CLI command:
-  - `dcld query pki revoked-noc-x509-root-cert --subject=<base64 string> --subject-key-id=<hex string>`
+  - `dcld query pki x509-cert --subject=<base64 string> --subject-key-id=<hex string>`
 - REST API:
-  - GET `/dcl/pki/revoked-noc-root-certificates/{subject}/{subject_key_id}`
+  - GET `/dcl/pki/certificates/{subject}/{subject_key_id}`
+
+#### GET_REVOKED_CERT
```

Comment:
Please clarify that it works for all types of certificates (PAA, PAI, NOC_ROOT, NOC_ICA).

Reply:
Done